development/build/static.js

```diff
@@ -127,11 +128,14 @@ function getCopyTargets(shouldIncludeLockdown) {
       dest: `vendor`,
     },
     {
-      src: `./node_modules/@fortawesome/fontawesome-free/webfonts/`,
+      src: getPathInsideNodeModules(
+        '@fortawesome/fontawesome-free',
+        'webfonts/',
+      ),
       dest: `fonts/fontawesome`,
     },
     {
-      src: `./node_modules/react-responsive-carousel/lib/styles`,
+      src: getPathInsideNodeModules('react-responsive-carousel', 'lib/styles'),
```

Comment:
above we do `images/` - any reason not to be consistent with trailing `/`?

Reply:
Yeah good call